administration/src/bp-modules/stores/StoresCSVInput.tsx

```diff
@@ -40,6 +41,16 @@ const lineToStoreEntry = (line: string[], headers: string[], fields: StoreFieldC
   return new AcceptingStoreEntry(storeData, fields)
 }
 
+const hasStoreDuplicates = (stores: AcceptingStoreEntry[]) => {
+  return (
+    new Set(
+      stores.map(({ data }: { data: StoreData }) =>
+        JSON.stringify([data['name'], data['street'], data['houseNumber'], data['postalCode'], data['location']])
```

Comment:
a random thought during testing:
if we have this list of fields to define duplicates, why don't we use these fields as keys to update acceptance points instead of recreating them?

Reply:
good point but if a store moved the location it will be even recreated. so a fixed id would be better